src/map.rs

```diff
@@ -1161,21 +1168,21 @@ where
     }
 
     fn add_count(&self, n: isize, resize_hint: Option<usize>, guard: &Guard) {
-        // TODO: implement the Java CounterCell business here
```

Comment:
There's a _lot_ more to [the real Java implementation of `CounterCell`](https://github.com/jonhoo/flurry/blob/13db2fb33d0c7ab45205ac5bb00d9b8749d959a2/jsr166/src/ConcurrentHashMap.java#L2295). We don't need to do all of that here in the current PR of course, but may be worth keeping a TODO around to point at the work that still remains.